src/services/print.js

```diff
@@ -214,6 +220,15 @@ ngeo.Print.prototype.encodeMap_ = function(map, scale, object) {
   let layers = this.ngeoLayerHelper_.getFlatLayers(mapLayerGroup);
   layers = layers.slice().reverse();
 
+  // (1.) We loop on every layers and if one of them doesn't use native angle,
+  // this set the global value to use for the print (default is true)
+  layers.forEach((layer) => {
+    if (layer.getProperties()['useNativeAngle'] === false) {
+      this.useNativeAngle_ = false;
+    }
+  });
+
```

Comment:
If one layer need it, it's active for every print request (layer groups).

I did this because I had an issue if I was mixing true and false between various layer groups (the layer was not visible once printed)... EPFL doesn't need to support this use case anyways but I don't want the print to break if some project just apply the layer.setProperties('useNativeAngle': false) to a single layer group.